client/tsconfig.json

```diff
@@ -76,13 +76,13 @@
     /* Type Checking */
     "strict": true,                                      /* Enable all strict type-checking options. */
     "noImplicitAny": false,                            /* Enable error reporting for expressions and declarations with an implied `any` type.. */
-    "strictNullChecks": false,                         /* When type checking, take into account `null` and `undefined`. */
+    "strictNullChecks": true,                         /* When type checking, take into account `null` and `undefined`. */
     // "strictFunctionTypes": true,                      /* When assigning functions, check to ensure parameters and the return values are subtype-compatible. */
     // "strictBindCallApply": true,                      /* Check that the arguments for `bind`, `call`, and `apply` methods match the original function. */
     // "strictPropertyInitialization": true,             /* Check for class properties that are declared but not set in the constructor. */
-    "noImplicitThis": false,                           /* Enable error reporting when `this` is given the type `any`. */
+    "noImplicitThis": true,                           /* Enable error reporting when `this` is given the type `any`. */
     // "useUnknownInCatchVariables": true,               /* Type catch clause variables as 'unknown' instead of 'any'. */
-    "alwaysStrict": false,                             /* Ensure 'use strict' is always emitted. */
+    "alwaysStrict": true,                             /* Ensure 'use strict' is always emitted. */
```

Comment:
You can change these back to `false` and leave them commented out to make our lives easier later.